sitemap-generator.js

```diff
@@ -1,12 +1,24 @@
 const sitemap = require('nextjs-sitemap-generator');
 const range = require('lodash/range');
+const constants = require('./src/utils/const');
 
 const chapters = range(1, 115);
 
+const generateVersesPaths = () => {
+  const paths = [];
+  range(114).forEach((chapterId) => {
+    const chapterNumber = chapterId + 1;
+    range(constants.CHAPTERS_DETAILS[chapterId].versesCount).forEach((verseId) => {
```

Comment:
I do like having this sort of data be in repo, so that we wouldn't have to wait for an api call, though it shouldn't be living in `constants` 

Will follow-up with an alternate solution